packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -26,8 +28,24 @@ export interface MenuProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'r
   'aria-label'?: string;
   /** Indicates if menu contains a flyout menu */
   containsFlyout?: boolean;
+  /** Indicates if menu contains a drilldown menu */
+  containsDrilldown?: boolean;
+  /** Indicates if a menu is drilled into */
+  isMenuDrilledIn?: boolean;
+  /** Indicates the drilldown path of menu itemIds */
+  drilldownPath?: string[];
+  /** Indicates the menus that are drilled in */
```

Comment:
I think the description is a little confusing.  May just "array of menus that are drilled into".